src/js/components/Widgets/CopyLinkModal.jsx

```diff
@@ -54,6 +54,8 @@ export default class CopyLinkModal extends Component {
               readOnly="true"
               value={urlBeingShared}
               className="form-control"
+              show={show}
+              onHide={onHide}
```

Comment:
We want these show and onHide props passed into "<Modal" (as opposed to passing into this "<input".